Tasks/Zlenko/Task1/depositCalc.py

```diff
@@ -0,0 +1,12 @@
+interestRates = [5, 10, 15, 20]
+years = [1, 3, 5, 10]
+periods = [1, 3, 12]
+
+principalAmount = int(input("Please enter your initial deposit amount:"))
+userInterestRate = int(input(f"Please select annual interest % rate from the list {interestRates}:"))/100
+userYears = int(input(f"Please select a number of years to keep the deposit in the bank {years}:"))
+userPeriods = int(input(f"Please select a number of compound interest periods per year {periods}:"))
```

Comment:
By adapting these values for the int type and using them in the math expression you get the float number as answer, which tends to accumulate some error, it's better to use decimal for money-related operations, -10 points for the logical mistake